shared/chat/conversation/input-area/user-mention-hud/index.js.flow

```diff
@@ -0,0 +1,45 @@
+// @flow
```

Comment:
why do we need this file? There's no cross-platform stuff going on....

Reply:
this is cause the typing of the composed components is totally messed up. flow really isn't inferring correctly whats going on. In order to make this work we'd have to really ensure the typing is good all the way up and down and it really isn't right now (sadly). Jacob spent a bunch of time on this and worked with me and right now its not worth it. But ideally it would.

Reply:
Ah, ok. Jacob can you add a comment/TODO here explaining what's going on?

Reply:
@akalin Yes totally. I wanted to get some feedback on the typing approach before commenting, now that's we're all aligned on this, I'll add some comments! 😄